mm2src/mm2_main/src/mm2.rs

```diff
@@ -278,15 +278,15 @@ pub fn mm2_main(version: String, datetime: String) {
 /// Parses and returns the `first_arg` as JSON.
 /// Attempts to load the config from `MM2.json` file if `first_arg` is None
 pub fn get_mm2config(first_arg: Option<&str>) -> Result<Json, String> {
-    let conf_path = env::var("MM_CONF_PATH").unwrap_or_else(|_| "MM2.json".into());
+    let conf_path = common::kdf_config_file();
```

Comment:
what about backward compatibility ?

Reply:
It is backward compatible: https://github.com/KomodoPlatform/komodo-defi-framework/blob/12620b28869fd313f311c7092f5fa56f2f3d88fc/mm2src/common/common.rs#L815-L819